README.md

```diff
@@ -301,25 +302,7 @@ You can download and run pre-built versions of GATK4 from the following places:
 * Note: Some GATK Spark tools by default require the reference file to be in 2bit format (notably `BaseRecalibratorSpark`,`BQSRPipelineSpark` and `ReadsPipelineSpark`). You can convert your fasta to 2bit by using the `faToTwoBit` utility from [UCSC](http://hgdownload.soe.ucsc.edu/admin/exe/) - see also the [documentation for `faToTwoBit`](https://genome.ucsc.edu/goldenpath/help/blatSpec.html#faToTwoBitUsage).
 
 #### <a name="R">Using R to generate plots</a>
-Certain GATK tools may optionally generate plots if R is installed.  We recommend **R v3.2.5** if you want to produce plots.  If you are uninterested in plotting, R is still required by several of the unit tests.  Plotting is currently untested and should be viewed as a convenience rather than a primary output.
-
-R installation is not part of the gradle build.  See http://cran.r-project.org/ for general information on installing R for your system.
-* for ubuntu see these [ubuntu specific instructions](http://cran.r-project.org/bin/linux/ubuntu/README)
-* for OSX we recommend installation through [homebrew](http://brew.sh/)
-```
-brew tap homebrew/science
-brew install R
-```
-
-The plotting R scripts require certain R packages to be installed. You can install these by running `scripts/docker/gatkbase/install_R_packages.R`.  Either run it as superuser to force installation into the sites library or run interactively and create a local library.
-```
-sudo Rscript scripts/docker/gatkbase/install_R_packages.R
-```
-**or**
-```
-R 
-source("scripts/docker/gatkbase/install_R_packages.R")
-```
+Certain GATK tools may optionally generate plots using the R installation provided within the conda environment.  If you are uninterested in plotting, R is still required by several of the unit tests.  Plotting is currently untested and should be viewed as a convenience rather than a primary output.
```

Comment:
Perhaps we should leave the "R" tag on the R tests and and change the build.gradle tests to run the "R" tests under the python package? That is probably more intuitive if in the future we want to reenable support for R.

Then under build.gradle (and dockertest.gradle) I would update the line:
```} else if (TEST_TYPE == "python") {
            includeGroups "python"
``` 
to be 
```} else if (TEST_TYPE == "python") {
            includeGroups "python", "R"
``` 
then further down exclude them by default. This is probably clearer as to what is going on. Then here I would add a comment that the unit tests requiring R are disabled by default and to activate them you must run the python tests.

Reply:
Added "conda" to the test matrix, which runs both "python" and "R" test groups, and changed back the "R" tag as you suggested.